en/overrides.md

```diff
@@ -30,6 +30,46 @@ To remove all current overrides on a channel:
 
     sudo snap-proxy delete-override <snap> <channel>
 
+### Revisions and Architectures
+
+Every *.snap file has a unique
+[revision](https://snapcraft.io/docs/getting-started) within a channel.
+
+One revision can support one or multiple architectures. Specifying a revision
+for an override will therefore determine for which architectures the override is
```

Comment:
A more usual phrasing/word order would be "Specifying a revision for an override therefore also determines the architectures involved."